src/data-source.validator.ts

```diff
@@ -1,7 +1,7 @@
 import { OpenApiDocument, OpenApiValidator } from 'express-openapi-validate';
 import { readSyncYaml } from './lib/read-sync-yaml';
 
-const specPath = __dirname + '/../spec/fdk-harvest-admin.yaml';
+const specPath = __dirname + '/spec/fdk-harvest-admin.yaml';
```

Comment:
Unnecessary to move spec over to src if you use [app-root-path](https://www.npmjs.com/package/app-root-path) (I can see you have removed it from package.json, please re-add it)

Also, use `path.join` to build dirpath instead of concatenation for code clarity and portability across OSes.

```
import path from 'path';
import { path as appRoot} from 'app-root-path';

const specPath = path.join(appRoot, 'spec', config.get('openapi.filename'); // I can see you have removed this config property. Please re-add it to default.json.
```

Reply:
fixed